src/main/scala/com/modelfabric/sparql/stream/client/Builder.scala

```diff
@@ -138,6 +139,7 @@ object Builder {
       import GraphDSL.Implicits._
       import endpoint._
 
+      // JC: why all stages in sparqlQueryFlow are async, but not here? do we need to make all of them async?
```

Comment:
I know that. Actually I have another question, is it necessary to make each single stage async. yeah, we'll need to find the best combination